src/sentry/mail/adapter.py

```diff
@@ -88,6 +85,24 @@ def get_digest_option(key):
 
         logger.info("mail.adapter.notification.%s" % log_event, extra=extra)
 
+    def active_release_notify(
```

Comment:
This basically follows the logic of `rule_notify` above, but removes all un-needed parameters related to rules (since rules won't be evaluated when this method gets called).

Reply:
nit: Is there a way we could DRY this?